src/unix/linux_like/emscripten/mod.rs

```diff
@@ -16,7 +16,21 @@ pub type loff_t = i64;
 pub type pthread_key_t = ::c_uint;
 
 pub type clock_t = c_long;
+
+// time_t was type:
+// 
+//   - long   from v1.5.4  (2013/08/08)  until v3.1.11 
+//   - int    from v3.1.11 (2022/05/20)  until v3.1.16 
+//   - _Int64 from v3.1.16  (2022/07/14)
+// 
+//  And it's still int64 today. The change from long to int was marked as a
+//  nonfunctional change since wasm64 was considered to be not working yet in
+//  v3.1.11. So it suffices to say it's 32 bit until v3.1.16 and 64 bit after.
```

Comment:
@sbc100 does this look correct to you?

Reply:
That looks right to me yes.

Reply:
(Hey Sam, long time! Hope you're doing well!)

Reply:
What!  Hi.   What a pleasure to see your name pop up in my inbox.   I hope you are doing well too.